cli/commands/migrate_create.go

```diff
@@ -175,13 +196,27 @@ func (o *migrateCreateOptions) run() (version int64, err error) {
 		}
 	}
 
-	if !o.flags.Changed("sql-from-file") && !o.flags.Changed("metadata-from-file") && !o.metaDataServer && !o.sqlServer && o.EC.Config.Version == cli.V1 {
+	// create pure sql based migrations here
+	if o.flags.Changed("up-sql") && o.flags.Changed("down-sql") {
+		errUpSQL := createOptions.SetSQLUp(o.upSQL)
```

Comment:
Handling error as soon as it happens will probably be better, something like
```
err = createOptions.SetSQLUp(o.upSQL)
if err != nil {
    return 0, errors.Wrap(errUpSQL, "up migration with SQL string could not be created")
}
```

Reply:
Cool! Will change that